wordle-web/components/keyboard.vue

```diff
@@ -146,3 +146,7 @@ export default class KeyBoard extends Vue {
   }
 }
 </script>
+
+<style scoped>
+
+</style>
```

Comment:
to del with buttons overlapping on smaller screen sizes you can try adding this to the `<v-btn>` section `:x-small="$vuetify.breakpoint.smAndDown"`